Harvest.Web/ClientApp/src/Expenses/UnbilledExpensesContainer.tsx

```diff
@@ -124,7 +125,10 @@ export const UnbilledExpensesContainer = (props: Props) => {
             )}
           </div>
           <div className="col text-right">
-            <ShowFor roles={["FieldManager", "Supervisor", "Worker"]}>
+            <ShowFor
+              roles={["FieldManager", "Supervisor", "Worker"]}
+              condition={project?.status !== "PendingCloseoutApproval"}
```

Comment:
Why `project?.status`?  We've already checked to see that project !== undefined or else we wouldn't be showing any of this html at all

Reply:
Didn't notice the `return <div>Loading...</div>;` part.